ssh/tailssh/tailssh.go

```diff
@@ -80,10 +90,20 @@ type server struct {
 	lb             *ipnlocal.LocalBackend
 	logf           logger.Logf
 	tailscaledPath string
+
+	// mu protects activeSessions.
+	mu             sync.Mutex
+	activeSessions map[string]bool
 }
 
 var debugPolicyFile = envknob.String("TS_DEBUG_SSH_POLICY_FILE")
 
+// portForward reports whether the ctx should be allowed to port forward
+// to the specified host and port.
+func (srv *server) portForward(ctx ssh.Context, _ string, _ uint32) bool {
```

Comment:
give these _ names. the func is small enough to see they're not used. I want to know what I could be using to make the decision.